src/context.ts

```diff
@@ -98,7 +98,7 @@ export class Context {
   }
 
   get me() {
-    return this.options.username
+    return this.botInfo?.username
```

Comment:
```suggestion
    return this.botInfo.username
```

Reply:
I agree it would be correct. It is what I wrote at first, too. But it breaks the tests because we never set the `botInfo` there, and this was the easiest solution, though QND.

Reply:
Actually type is correct, so whatever

Reply:
Yes, exactly my thinking :smile: